cluster/addons/storage-crds/csinodeinfo.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: csinodeinfos.csi.storage.k8s.io
-  labels:
+    labels:
```

Comment:
This looks wrong?

Reply:
@mikedanese resolved, thanks for the catch!